statistics/analyze_jobs.go

```diff
@@ -90,6 +91,12 @@ func GetAllAnalyzeJobs() []*AnalyzeJob {
 	return jobs
 }
 
+// GetChangedAnalyzeJobs returns all changed analyze jobs after last access.
+func GetChangedAnalyzeJobs() []AnalyzeJob {
+	// TODO: complete this function
```

Comment:
To keep this PR small and easy to review, this function will be completed in the next PR.